releasenotes/notes/structured-logging-26fa8f55ff5196b4.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    Add possibility to add structured logging (KV) to Agent logging system through `<log_level>c` functions
```

Comment:
(nit: this should be a user-facing release note, not a developer one)

Reply:
I tried to improve it in this PR: https://github.com/DataDog/datadog-agent/pull/6397